app/views/providers/saml_idp/idp/saml_post.html.erb

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge,chrome=1'>
+  </head>
+  <body onload="document.forms[0].submit();">
+    <%= form_tag("#{Rails.configuration.x.application.host_url}/providers/saml/auth") do %>
+      <%= hidden_field_tag('SAMLResponse', @saml_response) %>
+      <%= hidden_field_tag('RelayState', params[:RelayState]) %>
+      <%= submit_tag 'Submit' %>
+    <% end %>
+  </body>
+</html>
```

Comment:
Did you mean to add this file? I'm not sure what it's doing! If not you'll need to delete it

Reply:
Hi Mark. No, I didn't. I didn't touch that file but it somehow ended up as part of the commit. Where do I delete it?

Reply:
In your editor you should be able to see the file at app/views/providers/saml_idp/idp/saml_post.html.erb. You can just delete it from there. Then just commit again and push.

Reply:
OK (I think) I've done that